docs/source/using/experimental-features.adoc

```diff
@@ -152,6 +152,170 @@ To get the status of the DNS server:
 $ minishift dns status
 ----
 
+[[local-dns-setup-macos]]
+== Local DNS Setup for macOS
+
+Recent versions of macOS do not send out DNS queries in offline mode, and the process for using a local DNS server from {project} is more involved than other operating systems.
+
+=== Enable tap devices
+
+Check for the presence of `tap` devices in *_/dev_*:
+
+----
+$ ls /dev | grep tap
+----
+
+If no `tap` devices are present, install the *tuntap* package:
+
+----
+$ brew install tuntap
+----
+
+=== Use a tap device to create a network service
+
+As root, open the *_/Library/Preferences/SystemConfiguration/preferences.plist_* file and add the following XML under the `<key>NetworkServices</key>` element:
+
+----
+<key>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</key>	//<1>
+<dict>
+    <key>DNS</key>
+    <dict />
+    <key>IPv4</key>
+    <dict>
+        <key>Addresses</key>
+        <array>
+            <string>10.10.90.1</string>		//<2>
```

Comment:
convert to using `<!-- // 1 -->` as comments as those would NOT damage the actual XML if not correctly implemented.

Reply:
Good catch. XML comments will also continue to work for callouts.

Reply:
@robin-owen It does not work, the rendering is messed up it appears as normal text.. no red round button thingy anymore.
![screenshot from 2018-06-13 23-34-32](https://user-images.githubusercontent.com/8885742/41369390-5bb38b0e-6f62-11e8-952c-8c064b1af25f.png)

Anyways when it is _rendered_ if one wants to select the xml snippet only the text is selected without the callouts (the numbers), so i don't see a problem.. what am i missing?

/cc @gbraad

Reply:
@anjannath XML comments should work for callouts in AsciiDoc. Try changing the format of the comments to `<!--1-->`, `<!--2-->`, and so on. :)

Reply:
@anjannath is correct, when I copied the XML from the rendered doc the numbers did not come. Here is the copied content
```
<key>D16F22CE-6DDE-4E63-837C-E16538EA5CCB</key>	
<dict>
    <key>DNS</key>
    <dict />
    <key>IPv4</key>
    <dict>
        <key>Addresses</key>
        <array>
            <string>10.10.90.1</string>		
        </array>
        <key>ConfigMethod</key>
        <string>Manual</string>
        <key>SubnetMasks</key>
        <array>
            <string>255.255.0.0</string>
        </array>
    </dict>
    <key>IPv6</key>
    <dict>
        <key>ConfigMethod</key>
        <string>Automatic</string>
    </dict>
    <key>Interface</key>
    <dict>
        <key>DeviceName</key>
        <string>tap0</string>			
        <key>Hardware</key>
        <string>Ethernet</string>
        <key>Type</key>
        <string>Ethernet</string>
        <key>UserDefinedName</key>
        <string>MiniTap</string>		
    </dict>
    <key>Proxies</key>
    <dict>
        <key>ExceptionsList</key>
        <array>
            <string>*.local</string>
            <string>169.254/16</string>
        </array>
        <key>FTPPassive</key>
        <integer>1</integer>
    </dict>
    <key>SMB</key>
    <dict />
    <key>UserDefinedName</key>
    <string>MiniTap</string>			
</dict>
`

Reply:
So I am fine with current format.

Reply:
@robin-owen Do you know how to have the callouts as XML comment and still get it to render properly?